tests/plugins/ajax/ajax.js

```diff
@@ -179,6 +179,27 @@
 				} );
 			} );
 
+			wait();
+		},
+
+		// (#1134)
+		'test load async arraybuffer': function() {
```

Comment:
We are also missing `text` and `xml` types. I guess they are probably tested already as we are using them indirectly in ajax logic, but it would be good to check if API works correctly.